src/rendervulkan.cpp

```diff
@@ -324,7 +378,7 @@ bool CVulkanTexture::BInit( uint32_t width, uint32_t height, VkFormat format, cr
 	
 	if ( pDMA != nullptr )
 	{
-		assert( pDMA->n_planes == 1 );
+		assert( pDMA->n_planes == 1 || tiling == VK_IMAGE_TILING_DRM_FORMAT_MODIFIER_EXT );
```

Comment:
Actually, we still only import the first plane here, so we still need the assert `pDMA->n_planes == 1` regardless of the tiling.

Reply:
We actually import all planes at once here. (We can import planes with separate VkMemory objects only if the driver has DISJOINT support which e.g. radv doesn't and nobody in practice seems to need)

Reply:
Oh. But then this breaks if the DMA-BUF happens to have separate FDs for each plane?

Do we have a way to check whether a DMA-BUF requires DISJOINT imports?

Reply:
separate FDs is ok and expected, they're just supposed to point at the same buffer. That said, no idea how to check that besides opening a drm device, importing and checking whether you get the same handle ...

Reply:
Added a commit on top now that checks this.